osquery/logger/plugins/aws_kinesis.cpp

```diff
@@ -64,6 +82,9 @@ Status KinesisLogForwarder::send(std::vector<std::string>& log_data,
       LOG(ERROR) << "Kinesis log too big, discarding!";
     }
     Aws::Kinesis::Model::PutRecordsRequestEntry entry;
+    if (FLAGS_aws_kinesis_random_shardid) {
+      shard_id_ = random_string(32);
```

Comment:
haha i was going to use uuid but figured it seemed hacky.  I can switch to uuid.